File: library/src/main/java/cn/cqray/android/dialog/amin/BounceIn.java
package cn.cqray.android.dialog.amin;

import androidx.annotation.NonNull;

import cn.cqray.android.anim.AnimatorBuilder;

/**
 * 弹入
 * @author Cqray
 */
public class BounceIn extends DialogAnimator {
    @Override
    public void onHandle(@NonNull AnimatorBuilder builder) {
        builder.bounceIn();
    }
}

File: library/src/main/java/cn/cqray/android/dialog/amin/BounceIn.kt
package cn.cqray.android.dialog.amin

import cn.cqray.android.anim.ViewAnimator

/**
 * 弹入
 * @author Cqray
 */
class BounceIn : GetAnimator() {
    override fun onAnimatorPrepared(animator: ViewAnimator) {
        animator.bounceIn()
    }
}